### ISSUE TITLE
All the Backbone Net used by the compared methods are also frozen when training the datasets？

### ISSUE THREAD

**Issue by jchhuang** [zhmiao/OpenLongTailRecognition-OLTR#27]
Here， I am curious about that the weights of Backbone Net used by the compared methods are also frozen when training the datasets？If so，I think it is not fair to compare their performances since their performances does not exhaust if their weights are frozen

**Comment by zhmiao** [comment]
@jchhuang As written in the paper, in imagenet experiments where the weights are not frozen, they are not. In places experiments where the feature extractor weights are frozen, the other methods are frozen as well.

**Comment by jchhuang** [comment]
@zhmiao Thanks your reply. May you further explain why the weights of imagenet experiment are not frozen while the weights of Place are frozen？

**Comment by jchhuang** [comment]
@zhmiao If other methods are frozen as well, which means the performances of the compared methods are not exhausted, so the comparison results in your paper cannot indicate that your method outperform than others, does it?

**Comment by zhmiao** [comment]
@jchhuang As I said, in imagenet experiments, the weights of all methods are not frozen. In places experiments, the weights are frozen for all methods including ours. If that's not a fair comparison, I am not sure what is. Are you saying we should freeze the weights in our methods and not freeze the other methods? Is it fair? I am not sure what you are indicating.

**Comment by liuziwei7** [comment]
Using freezed network weights is equivalent to employ ImageNet pre-trained features, which is  a common practice in few/zero-shot learning community. For example, a closely related literature, "[Learning to Model the Tail](https://papers.nips.cc/paper/7278-learning-to-model-the-tail.pdf)", also adopted the freezed ResNet-152 as the backbone network for experimental comparisons.

Actually, this practice has an additional advantage that it separates the effect of feature and classifier, which could highlight the true merit of the classifier design (i.e. the desired information flow inside the classifier). 

Our open long-tailed recognition work serves as a start, instead of an ending point. You are more than welcomed to work on this topic and make contribution.
